addon/styles/_frost-guage.scss

```diff
@@ -0,0 +1,33 @@
+.frost-guage {
+  position: relative;
+
+  &-text {
```

Comment:
The `.selected` versus `-selected` I'm flexible on.  I definitely get your point about `-` being more secure in the global namespace, so I could get on board with that.  I'm used to "modifiers" being peer classes based on BEM (which focuses on scoping) but if you want to endorse `-` I'm game for that.

The last time we really documented CSS conventions was a year+ ago, so I'd be happy to do a new pass and solidify our strategy.